tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -78,6 +78,16 @@ cacl/test_cacl_application.py::test_multiasic_cacl_application:
     conditions:
       - "not is_multi_asic"
 
+cacl/test_cacl_application.py::test_cacl_scale_rules_ipv4:
+  skip:
+    reason: "caclmgrd may crash after loading scale ipv4 cacl rules."
+    conditions: https://github.com/sonic-net/sonic-buildimage/issues/10883
+
+cacl/test_cacl_application.py::test_cacl_scale_rules_ipv6:
+  skip:
+    reason: "caclmgrd may crash after loading scale ipv6 cacl rules."
+    conditions: https://github.com/sonic-net/sonic-buildimage/issues/10883
+
```

Comment:
Please keep the test case name in alphabetic order.

Reply:
Done